### PR TITLE
Improve vertical behavior

### PR DESCRIPTION
**Pull request by pdpino**
* Add `hoursInDisplay` prop to select amount of hours shown in the screen
* Add `startHour` prop to set the initial hour to display
* Disable event-pressing if no handler (`onEventPress`) is passed
* Fix typo in README.md

### CODE REVIEW

**Comment by hoangnm** [comment]
Thank you for the nice improvements! Could you help to resolve the conflicts? I just merged another MR.

File: src/WeekView/WeekView.js [hunk 1]
```diff
@@ -42,16 +45,28 @@ export default class WeekView extends Component {
...
 
   generateTimes = () => {
+    const { hoursInDisplay } = this.props;
     const times = [];
-    for (let i = 0; i < TIME_LABELS_COUNT; i += 1) {
-      const minutes = i % 2 === 0 ? '00' : '30';
-      const hour = Math.floor(i / 2);
-      const time = `${hour}:${minutes}`;
-      times.push(time);
+    const timeLabelsPerHour = TIME_LABELS_IN_DISPLAY / hoursInDisplay;
```
Comment (hoangnm) [inline src/WeekView/WeekView.js:62]: This improvement is nice, but the props name `hoursInDisplay` is kind of confusing to me. Could we have another name?

Comment (pdpino) [reply src/WeekView/WeekView.js:62]: Sure! any suggestions? I can think of `hoursInScreen`, `displayedHours`, `agendaLength`, `agendaHours`, but they may be confusing as well.
Any other ideas are welcomed

Comment (hoangnm) [reply src/WeekView/WeekView.js:60]: I can't think of a better name. We can improve later. Thanks again for the MR! I'm merging.

**Pull request merged**